README.md

```diff
@@ -7,27 +7,32 @@ for running a Minecraft server in a [dyno](https://devcenter.heroku.com/articles
 
 ## Usage
 
-Create a [free ngrok account](https://ngrok.com/) and copy your Auth token. Then create a new Git project with a `eula.txt` file:
+Create a [free ngrok account](https://ngrok.com/) and copy your Auth token. Then follow these instructions:
 
 ```sh-session
 $ echo 'eula=true' > eula.txt
+$ echo 's3cmd' > requirements.txt
```

Comment:
this file is already in the Git repo, why do you need this?

Reply:
It's because some people reported problems with this.

Reply:
Second this - looks like the Python buildpack fails on not finding [any expected files](https://devcenter.heroku.com/articles/deploying-python#expected-files-for-python). This is before this buildpack is hit, so I'm assuming that's why it doesn't see the `requirements.txt` file 🤔